management/src/test/java/org/kaazing/gateway/management/jmx/JmxRoundTripLatencyIT.java

```diff
@@ -39,7 +39,7 @@
 import org.kaazing.gateway.server.test.config.builder.GatewayConfigurationBuilder;
 import org.kaazing.k3po.junit.annotation.Specification;
 import org.kaazing.k3po.junit.rules.K3poRule;
-
```

Comment:
These test are consistently failing for me on windows with the same error mentioned here:   @Ignore("https://github.com/kaazing/tickets/issues/448")
    public void shouldKillSessionsByRolePrincipal() throws Exception 
I will revert the ignore as see if they pass on unix.